docs/detections/api/rules/rules-api-create-single-rule-exception-item.asciidoc

```diff
@@ -0,0 +1,86 @@
+[[exceptions-api-create-rule-default-exception-item]]
+=== Create exceptions for a rule 
+
+Allows you to create exception items that are associated with a specified rule `id`.
```

Comment:
My wording isn't a great, but maybe something like:

Adds specified exception items to a rules default exception list - if no default exception list exists, one is created and associated with the rule. Default exception lists are meant to house exceptions associated with a single rule, as opposed to shared exception lists which can be associated to multiple rules.

Reply:
Thank you! Added in [585ed2c](https://github.com/elastic/security-docs/pull/4021/commits/585ed2cb75a83d00203852718b0a7d5fb610f4f6).